src/server/MapPlaylist.ts

```diff
@@ -23,11 +23,32 @@ export class MapPlaylist {
     return mapsPlaylist.shift()!;
   }
 
+  private gameModeHistory: GameMode[] = [];
+
   public getNextGameMode(): GameMode {
-    const nextGameMode = this.gameModeRotation[this.currentGameModeIndex];
-    this.currentGameModeIndex =
-      (this.currentGameModeIndex + 1) % this.gameModeRotation.length;
-    return nextGameMode;
+    const FFA_WEIGHT = 2;
+    const TEAM_WEIGHT = 1;
```

Comment:
Is there any reason it will be teams only 1/3 of the time?
```suggestion
    const FFA_WEIGHT = 1;
    const TEAM_WEIGHT = 1;
```

Reply:
I believe this is how it was previously